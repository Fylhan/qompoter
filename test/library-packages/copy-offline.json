{
	"name": "install-ok/qompoter-versionstar-offline",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"cp/acme": "v1.*-lib",
		"cp/fylhan3": "v1.0-lib"
	}
}
