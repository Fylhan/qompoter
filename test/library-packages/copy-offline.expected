Qompoter
======== install -> create qompoter.lock

* cp/acme v1.*-lib
  Downloading lib...
  Selected version: v1.1-lib
  done

* cp/fylhan3 v1.0-lib
  Downloading lib...
  done

done
